app/views/partials/_about-us.html.haml

```diff
@@ -0,0 +1,6 @@
+.jumbotron.about-us-section
+  %h2.display-5 Abuelometro, está aquí para ayudar
+  %p.lead Somos una plataforma de monitoreo continuo para adultos mayores
+  %hr.my-4
```

Comment:
Pro Tip... whenever you naming classes, try and use something descriptive. This will help other whenever they join the project understand what they are being used for. I am guessing the `hr` here stands for `header-right` and the `my-4` mean something like `margin-y`? Create your application as clean and readable as possible so that anyone I mean anyone who has experience or doesn't can understand what you are trying to do.

Reply:
Those classes come directly from Bootstrap I use them to have faster development.